cmd/util/cmd/rollback-executed-height/cmd/rollback_executed_height.go

```diff
@@ -151,40 +155,42 @@ func removeExecutionResultsFromHeight(
 	myReceipts storage.MyExecutionReceipts,
 	events storage.Events,
 	serviceEvents storage.ServiceEvents,
-	fromHeight uint64) error {
+	fromHeight uint64) ([]flow.Identifier, error) {
```

Comment:
I prefer to put the return value on the next line. Thereby, the function body is visually separated from eh header via a line without indentation. It helps human brains to quicker structure the visually seen data according to broad structural information (color, indentation, etc) 
```suggestion
	fromHeight uint64,
) ([]flow.Identifier, error) {
```